swim/node.go

```diff
@@ -58,7 +58,9 @@ type Options struct {
 func defaultOptions() *Options {
 	opts := &Options{
 		StateTimeouts: StateTimeouts{
-			Suspect: 5000 * time.Millisecond,
+			Suspect:   5 * time.Second,
+			Faulty:    1 * time.Hour,
```

Comment:
I'd say the argument for keeping them is more to do with if we implement a partition-healing strategy based on faulty nodes in the future.

The reaping time defines the longest possible partition we can recover from with such a strategy.

My initial reaction is that longer is better then, for this case.

What are the chances that—depending on the infrastructure/environment—a new node might come online with the same host/port as a faulty node?